renderer/containers/cropper.js

```diff
@@ -77,8 +77,8 @@ export default class CropperContainer extends Container {
       displayId: id,
       x: x || screenWidth / 2,
       y: y || screenHeight / 2,
-      width: width || 0,
-      height: height || 0,
+      width: width || 400,
+      height: height || 400,
       ratio: ratio || [1, 1]
```

Comment:
I suggest setting the initial/default aspect ratio to 4:3, that aspect ratio works better across devices and services you're likely to upload to. 

```
width: width 400,
height: height 300,
ratio: ratio [4, 3]
```